### PR TITLE
Condensed syntax - inline keys support, optional blocks

### PR DESCRIPTION
**Pull request by gogiel**
Follow up to dead PR https://github.com/fotinakis/swagger-blocks/pull/27

Includes changes in spec.

### CODE REVIEW

File: lib/swagger/blocks.rb [hunk 1]
```diff
@@ -248,6 +249,10 @@ def data
         @data ||= {}
       end
 
+      def keys(values)
```
Comment (fotinakis) [inline lib/swagger/blocks.rb]: "values" sounds like a list, but it is in fact a dictionary. Please rename to "data"

**Comment by fotinakis** [comment]
I like this! Thanks for putting in the work. For some reason this seems to have broken Ruby 1.9.3 support, but that might be unrelated as we don't pin the `spec.add_development_dependency 'rspec'` version in gemspec.

One comment above and then I'll take another look.

**Comment by gogiel** [comment]
Thanks for the feedback.
I thought that Travis CI failure was related to https://github.com/bundler/bundler/issues/3558 I added bundler update to `.travis.yml` `before_install` block, but it failed with different error.

Travis tests [in my repo](https://travis-ci.org/4ormat/swagger-blocks/jobs/105797653) finished successfully with no changes to the code.
Maybe [clearing the cache](https://docs.travis-ci.com/user/caching/#Clearing-Caches) will help?

**Comment by fotinakis** [comment]
I've fixed the bundler error, if you pull from master it should work.

**Comment by gogiel** [comment]
Thank you, fixed!
Do you prefer rebase&squash before merging?

**Comment by fotinakis** [comment]
Yes please!

**Comment by gogiel** [comment]
:boom:

**Comment by fotinakis** [comment]
Released as gem v1.3.0.

Thanks for the contribution!

**Pull request merged**